### PR TITLE
Fix window.Store.ProfilePic.profilePicFind is not a function error

### LINKED ISSUE #3005: Unable to get conact avatar.
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

When trying to call `client.getProfilePicUrl` method `Error: Evaluation failed: TypeError: window.Store.ProfilePic.profilePicFind is not a function` will be thrown.

### Expected behavior

Get the url of the avatar of the contact.

### Steps to Reproduce the Bug or Issue

1. login.
2. get the avatar of yourself

### Relevant Code

Client.js
```ts
async getProfilePicUrl(contactId) {
        const profilePic = await this.pupPage.evaluate(async contactId => {
            try {
                const chatWid = window.Store.WidFactory.createWid(contactId);
                return await window.Store.ProfilePic.profilePicFind(chatWid);
            } catch (err) {
                if(err.name === 'ServerStatusCodeError') return undefined;
                throw err;
            }
        }, contactId);
        
        return profilePic ? profilePic.eurl : undefined;
    }
```

### Browser Type

Chromium

### WhatsApp Account Type

Standard

### Does your WhatsApp account have multidevice enabled?

Yes, I am using Multi Device

### Environment

OS: Mac
Node: 18
Whatsapp-web.js:  1.23.1-alpha.6
Whatsapp Web: 2.3000.1013249969

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by seowzhenjun0126**
# PR Details

Replace `window.Store.ProfilePic.profilePicFind` with `window.Store.ProfilePic.requestProfilePicFromServer`

## Description

The function

`window.Store.ProfilePic.profilePicFind`

is no longer available in v2.3x, and is replaced with

`window.Store.ProfilePic.requestProfilePicFromServer`

image

## Related Issue

closes #3005

## Types of changes

- [ ] Dependency change
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project.
- [ ] I have updated the documentation accordingly (index.d.ts).

### CODE REVIEW

**Comment by StiflerPasha** [comment]
and how about previous wa-versions?

**Comment by alechkos** [comment]
@StiflerPasha 
> and how about previous wa-versions?

In 2.2412.54, `window.Store.ProfilePic.profilePicFind` is defined

**Comment by StiflerPasha** [comment]
We need to make an implementation that is supported on both versions

**Review by alechkos — CHANGES_REQUESTED** [review]
@seowzhenjun0126 

Add support for previous WWeb versions,
you can use `window.WWebJS.compareWwebVersions` that is located in
`src/util/Injected/Utils.js`

**Review by guigo613 — CHANGES_REQUESTED** [review]
In this case, it will break for older versions. I believe the best solution would be something like:

```javascript
return await (window.Store.ProfilePic.profilePicFind || window.Store.ProfilePic.requestProfilePicFromServer)(chatWid);
```

**Review by N0N4M3BNS — APPROVED** [review]

**Comment by mauriciopfl** [comment]
> In this case, it will break for older versions. I believe the best solution would be something like:
> 
> ```js
> return await (window.Store.ProfilePic.profilePicFind || window.Store.ProfilePic.requestProfilePicFromServer)(chatWid);
> ```

Don`t work
Error on: window.WWebJS.compareWwebVersions

File: src/Client.js [hunk 1]
```diff
@@ -1209,7 +1209,7 @@ class Client extends EventEmitter {
         const profilePic = await this.pupPage.evaluate(async contactId => {
             try {
                 const chatWid = window.Store.WidFactory.createWid(contactId);
-                return await window.Store.ProfilePic.profilePicFind(chatWid);
+                return await (window.WWebJS.compareWwebVersions(window.Debug.VERSION, '<', '2.3000.0') ? window.Store.ProfilePic.profilePicFind(chatWid) : window.Store.ProfilePic.requestProfilePicFromServer(chatWid));
```
Comment (alechkos) [inline src/Client.js:1212]:
```suggestion
                return window.WWebJS.compareWwebVersions(window.Debug.VERSION, '<', '2.3000.0')
                    ? await window.Store.ProfilePic.profilePicFind(chatWid)
                    : await window.Store.ProfilePic.requestProfilePicFromServer(chatWid);
```

**Comment by Lyfhael** [comment]
Hey,
Should it be modified from this file too maybe ? https://github.com/pedroslopez/whatsapp-web.js/blob/41b280c769918db03ac6ec12cf0e4215e7b864ef/docs/Client.js.html#L1227

**Comment by erm3nda** [comment]
This has stopped to work for me few days ago, I don't know the exact moment. 
requestProfilePicFromServer function still exists but throws internal error evaluating e.isNewsletter and breaks.

**Pull request merged**